src/com/google/enterprise/adaptor/filenet/ConfigOptions.java

```diff
@@ -196,8 +195,16 @@ public ObjectStore getObjectStore(AutoConnection connection) {
     return objectFactory.getObjectStore(connection, objectStoreName);
   }
 
-  public String getDisplayUrl() {
-    return displayUrl;
+  public URI getDisplayUrl(Id objectId, Id versionSeriesId) {
```

Comment:
s/versionSeriesId/vsId/g: I think this is well-known enough, and it's consistent with DocumentTraverser.